### DISCUSSION TITLE
Default to color palette without signal colors for traces

### DISCUSSION THREAD

**Discussion by jeschkies** [grafana/grafana#37098] [Dashboards, Panels & Visualizations]
I've recently looked a trace visualization and was confused by the green, red and orange colored bars. They signaled and error or a warning to me. I was told the visualization was using the default Grafana color palette. It would be useful to default to one which does not include signal colors to avoid confusion.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
